pkg/common/constants.go

```diff
@@ -15,6 +15,7 @@ const (
 	NetworksPath = "networks"
 	// NetworkTypePath
 	NetworkTypePath = "network-types"
+	NetworkPoolPath = "pools"
```

Comment:
Should the CI fail here? or have we removed the CI validation for Commment on exported names?

Reply:
I will go with the second approch. It will be a huge task to comment all exported fields in sdk

Reply:
yeah, sure. But my concern is did the CI run fine?

Reply:
Yeah, we can explicitily specify the CI to skip the checks for exported names